miniforge-cuda/Dockerfile

```diff
@@ -96,7 +96,8 @@ RUN wget --quiet ${MINIFORGE_URL} -O /miniforge.sh \
 RUN conda install -k -y tini
 
 # Clean up conda and set permissions for all users
-RUN /opt/conda/bin/conda clean -tipsy \
+RUN chmod -R ugo+w /opt/conda\
```

Comment:
```suggestion
RUN chmod -R ugo+w /opt/conda \
```

Reply:
Yeah I'll update them, thanks for the catch